overpass/api.py

```diff
@@ -191,21 +192,49 @@ def _as_geojson(self, elements):
         geometry = None
         for elem in elements:
             elem_type = elem.get("type")
-            if elem_type and elem_type == "node":
+            elem_tags = elem.get("tags")
+            elem_geom = elem.get("geometry", [])
+            if elem_type == "node":
+                # Create Point geometry
                 geometry = geojson.Point((elem.get("lon"), elem.get("lat")))
-            elif elem_type and elem_type == "way":
-                points = []
-                geom = elem.get("geometry")
-                if geom:
-                    for coords in elem.get("geometry"):
-                        points.append((coords["lon"], coords["lat"]))
-                    geometry = geojson.LineString(points)
+            elif elem_type == "way":
+                # Create LineString geometry
+                geometry = geojson.LineString([(coords["lon"], coords["lat"]) for coords in elem_geom])
+            elif elem_type == "relation":
+                # Initialize polygon list
+                polygons = []
+                # First obtain the outer polygons
+                for member in elem.get("members", []):
+                    if member["role"] == "outer":
+                        points = [(coords["lon"], coords["lat"]) for coords in member.get('geometry', [])]
+                        # Check that the outer polygon is complete
+                        if points and points[-1] == points[0]:
```

Comment:
else?

Reply:
Not sure which line you mean here: 208 or 211?
208: I don't think we should put an else here. Also there are different relation types in Overpass (next to polygons) which are not all covered by this code here. But that's rather another improvement that should be covered in separate pull request. I would not throw an error about this to prevent that existing projects using this package fail.
211: It is fixed now.

Reply:
I only refered to 211. Thanks 👌